pages.ru/common/hexdump.md

```diff
@@ -5,16 +5,16 @@
 
 - Распечатать шестнадцатеричное представление файла, заменяя повторяющиеся строки на '*':
 
-`hexdump {{файл}}`
+`hexdump {{путь/до/файла}}`
```

Comment:
The [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/hexdump.md) page should be updated too to reflect this change.